src/commands/convert/msft_secure2hdf.ts

```diff
@@ -0,0 +1,130 @@
+import {Command, Flags} from '@oclif/core'
+import fs from 'fs'
+import {MsftSecureScoreMapper as Mapper} from '@mitre/hdf-converters'
+import {checkSuffix} from '../../utils/global'
+import {ClientSecretCredential} from '@azure/identity'
+import {Client, ClientOptions} from '@microsoft/microsoft-graph-client'
+import {
+  SecureScore,
+  SecureScoreControlProfile,
+} from '@microsoft/microsoft-graph-types'
+import {TokenCredentialAuthenticationProvider} from '@microsoft/microsoft-graph-client/authProviders/azureTokenCredentials'
+
+function processInputs(
+  scoreDoc: SecureScore,
+  profiles: { value: SecureScoreControlProfile[] },
+  output: string,
+  withRaw: boolean,
+): any {
+  const converter = new Mapper(
+    JSON.stringify({
+      secureScore: scoreDoc,
+      profiles: profiles,
+    }),
+    withRaw,
+  )
+  fs.writeFileSync(checkSuffix(output), JSON.stringify(converter.toHdf()))
+}
+
+export default class MsftSecure2HDF extends Command {
+  static usage = [
```

Comment:
usually prefer a single usage string where exclusive flags are delineated appropriately.  we try to use this as the standard for how to write usage strings: http://docopt.org/